draft-ietf-quic-multipath.md

```diff
@@ -254,7 +248,7 @@ When the multipath option is negotiated, clients that want to use an
 additional path MUST first initiate the Address Validation procedure
 with PATH_CHALLENGE and PATH_RESPONSE frames described in
 {{Section 8 of QUIC-TRANSPORT}}. After receiving packets from the
-client on the new paths, the servers MAY in turn attempt to validate
+client on the new paths, the servers MUST in turn attempt to validate
```

Comment:
This seems more related to the discussion in #101, not sure if it should be part of this PR.

Reply:
Reverted.